dbcrossbarlib/src/drivers/postgres_shared/data_type.rs

```diff
@@ -0,0 +1,272 @@
+//! PostgreSQL data types.
+
+use std::fmt;
+
+use crate::common::*;
+use crate::schema::DataType;
+
+/// An SRID number specifying how to intepret geographical coordinates.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub(crate) struct Srid(u32);
+
+impl Srid {
+    /// Create a new `Srid` from a numeric code.
+    pub(crate) fn new(srid: u32) -> Srid {
+        Srid(srid)
+    }
+
+    /// Return our `Srid` as a `u32`.
+    pub(crate) fn to_u32(self) -> u32 {
+        self.0
+    }
+}
+
+impl Default for Srid {
+    fn default() -> Self {
+        // The one true SRID.
```

Comment:
// WGS84, which measures things in degrees

Reply:
Added! Thank you.